wpilibj/src/main/java/edu/wpi/first/wpilibj/Timer.java

```diff
@@ -15,21 +15,37 @@
    * @return Robot running time in seconds.
    */
   @SuppressWarnings("AbbreviationAsWordInName")
-  public static double getFPGATimestamp() {
-    return RobotController.getFPGATime() / 1000000.0;
+  public static double getFPGATimestampSeconds() {
```

Comment:
Since this is a very widely used method, I believe the original method signature should be kept, but with a `@Deprecated` tag, similar to the `getMatchTimeSeconds()` and the deprecated `getMatchTime()` methods below.